ValidatorPanel.php

```diff
@@ -769,6 +769,11 @@ class ValidatorPanel implements Tracy\IBarPanel
 	 * @var array|\LibXMLError[]
 	 */
 	private $errors = array();
+	
+	/**
+	 * @var array
```

Comment:
I haven't setup any project to test this yet so I'm not sure what data type will be here. I guess it can be changed to `@var string[]`

Reply:
You are right. I fix this.